src/olympia/stats/tests/test_utils.py

```diff
@@ -236,3 +236,15 @@ def test_returned_results(self, bigquery_client_mock):
 
         returned_results = get_addons_and_average_daily_users_from_bigquery()
         assert returned_results == [(1, 123), (2, 456)]
+
+    @mock.patch('google.cloud.bigquery.Client')
+    def test_skips_null_values(self, bigquery_client_mock):
+        results = [
+            self.create_bigquery_row({'addon_id': 1, 'count': 123}),
+            self.create_bigquery_row({'addon_id': 2, 'count': None}),
+        ]
```

Comment:
isn't this testing the wrong thing? It's `None` values in the database rather the bigquery that are failing.

Reply:
BigQuery can return `count=None` and the DB can return `guid=None`

Reply:
We should probably have a test case for the DB change but simply adding an add-on without a GUID does not seem to work 🤷

Reply:
> 
> 
> BigQuery can return `count=None` and the DB can return `guid=None`

right, sure, but the change above is only related to the database query, but this test only (appears to) mock bigquery response.

Reply:
> 
> 
> We should probably have a test case for the DB change but simply adding an add-on without a GUID does not seem to work 🤷

what do you mean doesn't work?  You can't `.update(guid=None)`?

Reply:
Yeah, that works (or `addon_factory(guid=None)`) but for some reason, we don't get an error raised by the subtask in this cron task..

Reply:
I added another test case.